schedule/adapter.py

```diff
@@ -26,7 +26,7 @@ def course_to_course_offering(course: Course) -> A_CourseOffering:
 
 
 def add_course_offering_to_schedule(course: Course, a_course_offering: A_CourseOffering):
-    schedule, _ = A_Schedule.objects.get_or_create(id=0)
+    schedule, _ = A_Schedule.objects.get_or_create(id=1)
```

Comment:
One more thing regarding querying - If I delete the Schedule object from the Django admin page, that `id` is never recycled, so I'm forced to fully torch the DB and rebuild. Do we want to avoid this inconvenience?

If so, something like `A_Schedule.objects.first()` will always return the oldest object in the DB, so its `id` needs not be number1 specifically. Let me know what you think or if you have an idea